RSS Reader/RSS Reader/View Controllers/Topics list/View /RSSRTopicsListViewController.m

```diff
@@ -162,6 +194,18 @@ - (void)endRefreshing {
     [self.tableView.refreshControl endRefreshing];
 }
 
+
+#pragma mark - RSSRTopicCellDelegate Protocol
+
+- (void)reloadCellWithTopic:(id<RSSRTopicItemProtocol>)topic {
+    NSIndexPath *indexPath = [NSIndexPath indexPathForRow:[[self.presenter topics] indexOfObject:topic] inSection:0];
+    
+    [UIView performWithoutAnimation:^{
+            [self.tableView reloadRowsAtIndexPaths:@[indexPath]
```

Comment:
What for do you need this?

Reply:
There is a problem with animation when you click on the show/hide details button. It may not be the best solution but it works for me so far